app/models/public_body.rb

```diff
@@ -75,6 +75,9 @@ class ImportCSVDryRun < StandardError ; end
   after_update :reindex_requested_from
 
 
+  # Force eager loading of translations everywhere
+  default_scope { includes(:translations) }
```

Comment:
Maybe should be a proc? https://stackoverflow.com/questions/4731125/always-eager-load-association-with-entity#comment49457616_4731370

```diff
-  default_scope { includes(:translations) }
+  default_scope -> { includes(:translations) }
```